src/script/tracking/countly-skd-web.d.ts

```diff
@@ -41,6 +41,8 @@ export interface CountlyEvent {
 }
 
 export interface Countly {
+  q: any[];
```

Comment:
we dont have any expectations about this array?

Reply:
Countly has no types. We could implement all of them by ourselves, but this is a waste of time for me, since the library can change any time and our types will be outdated without our knowledge. 
Its better to open the countly documentation when working on it.

Reply:
maybe theres a link you could insert here then? to make it easier for the next person

Reply:
as a code comment.

Reply:
Oh, for sure. Good call